### PR TITLE
Issue template

### LINKED ISSUE #2347: Template for new Issues
Provide a github template for new issues

### PR DESCRIPTION
**Pull request by tdipisa**
## Description
This pull request provides the template to use for new issues 

## Issues
 - Fix #2347 

**What kind of change does this PR introduce?**
 - [x] Other: github template for new issues

**What is the current behavior?** 
#2347 

**What is the new behavior?**
Template available for new issues

**Does this PR introduce a breaking change?** 

 - [ ] Yes
 - [x] No

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Description
+A few sentences describing the overall goals of the issue.
+
+## Depends On
+ - A list of other involved issues (with link)
+ - ...
+
+**Please indicate if this issue is related to a bug or a new feature request**
+- [ ] Bug
```
Comment (mbarto) [inline .github/ISSUE_TEMPLATE.md]: We are already using labels for this.

Comment (tdipisa) [reply .github/ISSUE_TEMPLATE.md]: The manager will associate label properly

Comment (mbarto) [reply .github/ISSUE_TEMPLATE.md]: In general. why we repeat the same information in many different places?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+## Description
+A few sentences describing the overall goals of the issue.
+
+## Depends On
```
Comment (offtherailz) [inline .github/ISSUE_TEMPLATE.md]: Should we specify this is optional?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+*Browser Affected*
+
+- [ ] Internet Explorer
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Safari
+
+*Browser Version Affected*
+
+- Indicate the browser version in which the issue has been found
```
Comment (offtherailz) [inline .github/ISSUE_TEMPLATE.md]: We could link this site: https://www.whatsmybrowser.org/ for non expert users

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+## Description
+A few sentences describing the overall goals of the issue.
+
+## Depends On
+ - A list of other involved issues (with link)
+ - ...
+
+**Please indicate if this issue is related to a bug or a new feature request**
```
Comment (offtherailz) [inline .github/ISSUE_TEMPLATE.md]: a paragraph with ## notation can make it more readable

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- Describe here the current behavior
+
+**In case of New Feature (otherwise remove this paragraph)**
+
+*Acceptance Criteria - AC*
+
+- Describe the list of Acceptance Criteria
+- ...
+
+**Other useful information**:
```
Comment (MV88) [inline .github/ISSUE_TEMPLATE.md]: What about adding something like : 
(a screenshot / gif / a jsfiddle are welcome)

**Pull request merged**